### ISSUE TITLE
Cannot translate to multiple languages in a loop

### ISSUE THREAD

**Issue by swetabhmukherjee** [KhaledMohamedP/translate-json-object#35]
Hello @KhaledMohamedP, I am trying to translate JSON in 3 different languages with 3 outputs but am getting a totally absurd output. Can you suggest a fix?

**Comment by KhaledMohamedP** [comment]
Hi @swetabhmukherjee!  Can you provide more information, so that I can investigate on my end. Maybe a tiny example of what you want to achieve but couldn't?

**Comment by swetabhmukherjee** [comment]
I am trying to translate a json object in 3 different languages in one go. The code which I am using is shown below:
`
var languages = ["en","bn","hi"];
async function translateJSON(){
    
    for(var i=0;i<languages.length;i++){
        TJO.translate(JSON, languages[i])
        .then(function(data) {
        }).catch(function(err) {
          console.log('error translating', err)
        });
    }
}`

When I run this the parts of the JSON objects are translated in random languages. I expect an output as follows:
JSON 1
{
    My JSON in 'en' language
}
JSON 2
{
    My JSON in 'bn' language
}
JSON 3
{
    My JSON in 'hi' language
}

Can you tell how can I achieve this? any changes in the code?

P.S. My JSON looks something like this:

[
{
Question: "question1",
Option1: "option1",
Option2: "option2",
correct_answer: "correct answer"
},
Question: "question2",
Option1: "option1",
Option2: "option2",
correct_answer: "correct answer"
},
Question: "question2",
Option1: "option1",
Option2: "option2",
correct_answer: "correct answer"
}
]

**Comment by KhaledMohamedP** [comment]
There might be a bug in the code related to async calls, please try the following solution until I create a fix for this problem 

```js
for(lang of ["en","bn", 'hi', 'es']) {
    const result = await TJO.translate(example, lang)
    console.log(lang, result);
}
```

**Comment by KhaledMohamedP** [comment]
Should be fixed with latest version `2.3.7`

```
npm i translate-json-object@latest
```
